pkg/cmd/issue/create/create.go

```diff
@@ -157,6 +157,10 @@ func createRun(opts *CreateOptions) error {
 		return fmt.Errorf("must provide --title and --body when not attached to a terminal")
 	}
 
+	if interactive && !opts.IO.CanPrompt() {
+		return fmt.Errorf("must provide --title and --body when prompts are disabled")
```

Comment:
Nit: if both of these checks are only dependent on the state of `--title` and `--body` as provided by flags, could we move them to the flag-parsing section, e.g. the RunE block? This would ensure that we error out as soon as we have the information that the command cannot proceed